src/app/garfile/manifest/index.njk

```diff
@@ -187,16 +187,34 @@
         <input class="govuk-input govuk-input--width-3" id="totalPersonsOnBoard" name="name" readonly type="number" value="{{ manifest.items.length }}">
       </div>
 
-      <input id="save-and-continue" class="govuk-button" type="submit" name="buttonClicked" value="Save and continue">
+    {# Start buttons #}
```

Comment:
Manifest screen used to have notions of "Save and continue" which does not apply, so changed to "Continue"